### DISCUSSION TITLE
Upscale or larger original image

### DISCUSSION THREAD

**Discussion by mkfoto** [lllyasviel/Fooocus#1462] [General]
Hi folks,

been using fooocus for a few days now and I love it! Everything about it! :D Well, almost everything..
Usually the default image size is ok for me, but sometimes I would like a bit bigger image. So I've used upscaler which works BUT it also changes the image. Not by much, but it does. When you're working with faces or images with a lot of details even small changes can make a big difference. 
So my question, finally, is there a way to "fix" upscaler not to change image at all and just upscale it? Or is there a way to add a bigger size to the original aspect ratios prompt when generating original images?

Thanks,
Mat

**Comment by mkfoto** [comment]
I did find that you can do some changes via config file, but with mixed resault. Would still love to "fix" the image and use upscaler, so no changes are made. IF that's even possible...

**Comment by mashb1t** [comment]
You can use the "Upscale (Fast 2x)" to only do upscaling without SD generation.
See https://github.com/lllyasviel/Fooocus/discussions/390

**Reply by mkfoto** [reply]
True, but the image lacks a lof of details, it's not the same as the original one.. :/
